doc/website/blog/tmp-versioned-migrations-sum-file.md

```diff
@@ -0,0 +1,249 @@
+---
+title: Versioned Migrations Management and Migration Directory Integrity File
```

Comment:
```suggestion
title: Versioned Migrations Management and Migration Directory Integrity
```

Feels a bit long. maybe the "File" can be omitted because there will be other ways for ensuring integrity (replaying migration history).